Lesson7_OOPS.cpp

```diff
@@ -0,0 +1,106 @@
+/* DESIGN A VENDING MACHINE */
+#include<iostream>
+using namespace std;
+class Inventory
+{
+	int coke,sprite,chips;
+	static int coke_count,sprite_count,chips_count;
+	public:
+	Inventory()
+	{
+		coke=10;
+		sprite=15;
+		chips=20;
+	}
+	int getPrice(int item_num)
+	{
+		switch(item_num)
+		{
+			case 1: return coke;
+			case 2: return sprite;
+			case 3: return chips;
+			default: return -1;
+		}
```

Comment:
instead of using an int, you could use an enum if you have a fixed set of items, otherwise, I think it would be better to have some kind of item class and a map itemId -> item, then you could just use item.price

Reply:
should the default  case be an exception?